src/init.ts

```diff
@@ -160,8 +249,9 @@ export async function init() {
 
   const newConfig = {
     ...existingConfig,
-    mcpServers: { ...existingConfig.mcpServers, ...config },
+    [configKey]: { ...existingConfig[configKey], ...config },
   };
+
```

Comment:
I'd prefer you made a backup of the configuration file before trying to write to it just in case you somehow corrupt or write invalid data shapes to the file you're editing.

Reply:
Done